pandas/core/series.py

```diff
@@ -2530,7 +2530,11 @@ def idxmin(self, axis: Axis = 0, skipna: bool = True, *args, **kwargs) -> Hashab
         nan
         """
         axis = self._get_axis_number(axis)
-        i = self.argmin(axis, skipna, *args, **kwargs)
+        with warnings.catch_warnings():
+            # ignore warning produced by argmin since we will issue a different
+            #  warning for idxmin
+            warnings.simplefilter("ignore")
```

Comment:
Will we be able to remove this in 3.0? (if so can you make a note)

Reply:
yes, will update